node/src/components/block_validator.rs

```diff
@@ -68,14 +67,14 @@ pub enum Event<T, I> {
     #[display(fmt = "deploy {} found", deploy_hash)]
     DeployFound {
         deploy_hash: DeployHash,
-        deploy_size: usize,
+        deploy_type: Box<DeployType>,
     },
 
     /// A request to find a specific deploy, potentially from a peer, failed.
     #[display(fmt = "deploy {} missing", _0)]
     DeployMissing(DeployHash),
 
-    /// Deploy was invalid. Failed the chainspec test.
+    /// Deploy was invalid. Unable to convert to a deploy type.
```

Comment:
Is `DeployInvalid` now meaning _only_ inability to convert to a deploy type? If so, then we should change the name of this enum variant.

Reply:
Yes, although failing to convert still means that it's invalid. Maybe `DeployConversionFailed` or `CouldNotCreateDeployType`?